go/vt/schemamanager/tablet_executor.go

```diff
@@ -248,7 +248,17 @@ func (exec *TabletExecutor) executeAlterMigrationThrottle(ctx context.Context, a
 		if throttlerConfig.ThrottledApps == nil {
 			throttlerConfig.ThrottledApps = make(map[string]*topodatapb.ThrottledAppRule)
 		}
-		throttlerConfig.ThrottledApps[req.ThrottledApp.Name] = req.ThrottledApp
+		if req.ThrottledApp != nil && req.ThrottledApp.Name != "" {
+			// TODO(shlomi) in v22: replace the following line with the commented out block
```

Comment:
Because a v20 tablet server is sadly not too smart here. It needs to be _told_ "this rule is now expired in the throttler config" or otherwise it will not remove it from its records. Whereas in v21 the tablet looks at the throttler config, sees what exists, and ensures to delete from its own records anything that does not exist in the config.